contrib/patterns/_grid.scss

```diff
@@ -0,0 +1,19 @@
+.grid {
+  @include grid-container;
+}
+
+.grid__column {
+  @include grid-column;
+}
+
+.grid__column--thirds {
+  @include grid-column(4);
+}
+
+.grid__column--thirds--alt {
```

Comment:
Ahh I don't even like my own `grid__column--thirds--alt` here. Really it's a different grid. maybe I'll pull it out